product/set_inventory.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2021 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START add_fulfillment_places]
+import datetime
+import os
+import time
+
+from google.api_core.client_options import ClientOptions
+from google.cloud.retail import ProductServiceClient, SetInventoryRequest, PriceInfo, FulfillmentInfo
+from google.cloud.retail_v2 import Product
+
+from setup_cleanup import create_product, get_product, delete_product
+
+project_number = os.getenv('PROJECT_NUMBER')
+endpoint = "retail.googleapis.com"
+product_id = "inventory_test_product_id"
+product_name = 'projects/' + project_number + '/locations/global/catalogs/default_catalog/branches/default_branch/products/' + product_id
+
+# get product service client
+def get_product_service_client():
+    client_options = ClientOptions(endpoint)
+    return ProductServiceClient(client_options=client_options)
+
+
+# product inventory info
+def get_product_inventory_info(product_name: str) -> Product:
```

Comment:
you return Product from get_product_inventory_info method.

Reply:
method renamed